### PR TITLE
Explanations for meta interactions

### PR DESCRIPTION
**Pull request by egekorkan**
Address the following issues:
- #845 
- #844 
- #843

***
<a href="https://pr-preview.s3.amazonaws.com/egekorkan/wot-thing-description/pull/853.html" title="Last updated on Dec 6, 2019, 7:55 AM UTC (814e945)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/853/a80bdd2...egekorkan:814e945.html" title="Last updated on Dec 6, 2019, 7:55 AM UTC (814e945)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3635,6 +3643,42 @@ <h4>Protocol Binding based on HTTP</h4>
                     <code>Form</code> with operation type <code>invokeaction</code>
                 </td>
             </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-http-method_readall">
```
Comment (sebastiankb) [inline index.html]: I would it keep more simple:  in the first row in the table I would extend in the last column "Form with operation type readproperty, readallproperties, or readmultipleproperties"

File: index.html [hunk 2]
```diff
@@ -3635,6 +3643,42 @@ <h4>Protocol Binding based on HTTP</h4>
...
+            <tr class="rfc2119-default-assertion" id="td-default-http-method_readmultiple">
+                <td>
+                    <code>htv:methodName</code>
+                </td>
+                <td><code>GET</code></td>
+                <td>
+                    <code>Form</code> with operation type <code>readmultipleproperties</code>
+                </td>
+            </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-http-method_writeall">
```
Comment (sebastiankb) [inline index.html]: same here: in the second row I would add: "Form with operation type  writeproperty, writeallproperties, or writemultipleproperties"

File: index.html (hunk 1 above)
Comment (egekorkan) [reply index.html]: I agree, you should see the change in the new commit

File: index.html (hunk 2 above)
Comment (egekorkan) [reply index.html]: I agree, you should see the change in the new commit

File: index.html [hunk 3]
```diff
@@ -2959,6 +2959,14 @@ <h3><code>forms</code></h3>
 In the case of operation type <code>writeallproperties</code>, it is expected that the <a>Consumer</a> provides ALL writable (non <code>readOnly</code>) properties and the (new) assigned values (e.g., within payload). Otherwise the <a>Thing</a> MAY refuse this call to avoid inconsistency.
 </span></p>
 
+<p><span class="rfc2119-assertion" id="td-writemultiple-consumer">
```
Comment (sebastiankb) [inline index.html]: we are not allowerd to create a new assertion. We should change this to a 'information' for developer.

**Comment by sebastiankb** [comment]
the group decided to merge this PR

**Pull request merged**